src/core/unit-serialize.c

```diff
@@ -590,8 +590,6 @@ static void print_unit_dependency_mask(FILE *f, const char *kind, UnitDependency
                 { UNIT_DEPENDENCY_DEFAULT,            "default"            },
                 { UNIT_DEPENDENCY_UDEV,               "udev"               },
                 { UNIT_DEPENDENCY_PATH,               "path"               },
-                { UNIT_DEPENDENCY_MOUNTINFO_IMPLICIT, "mountinfo-implicit" },
-                { UNIT_DEPENDENCY_MOUNTINFO_DEFAULT,  "mountinfo-default"  },
```

Comment:
Please do not drop them. These dependencies are serialized. If we drop them, then the dependencies built by the older systemd will be dropped on `systemctl daemon-reexec` and that may cause unexpected results.

Reply:
Sorry, I thought so, but they do not. Please ignore the comment. Sorry for the noise.